tasks/release.py

```diff
@@ -1741,6 +1741,7 @@ def _create_build_links_patterns(current_version, new_version):
     new_minor_version = new_version[1:]
 
     patterns[current_minor_version] = new_minor_version
+    patterns[current_minor_version[2:].replace("-rc", "~rc")] = new_minor_version[2:].replace("-rc", "~rc")
```

Comment:
I think it would be easier to understand if `current_minor_version` and `new_minor_version` would be updated before this line (to avoid doing 2 times the replace)

Reply:
I think it's fair, updated. Let me know if this is what you thought